README.md

```diff
@@ -41,6 +41,8 @@ It also has optional support for more formats via external libraries:
 
 * [JPEG](https://en.wikipedia.org/wiki/JPEG) via [IJG/libjpeg](http://www.ijg.org/) or [libjpeg-turbo](http://libjpeg-turbo.virtualgl.org/)
   * Does not include [JPEG 2000](https://en.wikipedia.org/wiki/JPEG_2000)
+* [HEIF](https://en.wikipedia.org/wiki/High_Efficiency_Image_File_Format) via [libheif](https://github.com/strukturag/libheif/)
```

Comment:
"H" before "J" :)

Reply:
Completely overlooked that, but it's :+1: now.